framework/src/utils/SlepcSupport.C

```diff
@@ -265,8 +265,15 @@ setSlepcOutputOptions(EigenProblem & eigen_problem)
       Moose::PetscSupport::setSinglePetscOption("-eps_power_ksp_monitor");
       break;
 
-    case Moose::EST_MF_MONOLITH_NEWTON: // This should be removed once RattleSnake is updated
-      /* Falls through. */
+    // This should be removed once Griffin is updated
+    case Moose::EST_MF_MONOLITH_NEWTON:
+      Moose::PetscSupport::setSinglePetscOption("-init_eps_monitor_conv");
+      Moose::PetscSupport::setSinglePetscOption("-init_eps_monitor");
+      Moose::PetscSupport::setSinglePetscOption("-eps_power_snes_monitor");
+      Moose::PetscSupport::setSinglePetscOption("-eps_power_ksp_monitor");
+      Moose::PetscSupport::setSinglePetscOption("-init_eps_power_snes_monitor");
+      Moose::PetscSupport::setSinglePetscOption("-init_eps_power_ksp_monitor");
+      break;
     case Moose::EST_NEWTON:
```

Comment:
This is an empty case statement; this is fine to "fallthrough" on. The compiler is not warning/erroring about this, is it?

Reply:
Yes, `case Moose::EST_MF_MONOLITH_NEWTON` should be empty in theory since it will take the same thing as` case Moose::EST_NEWTON:`

But complier is angry at me if I leave `case Moose::EST_MF_MONOLITH_NEWTON` empty 

https://civet.inl.gov/job/530953/

Reply:
We do not need to worry about this code. It will be removed once I get Griffin updated. But we need to get the moose changes in first

Reply:
the compiler is only warning about the below case; not this one

Reply:
But since it's temporary, I won't make you update the PR

Reply:
Yes, we do not need to worry about this temporary code. I will remove it soon